tests/page.py

```diff
@@ -0,0 +1,22 @@
+from bs4 import BeautifulSoup
+
+
+class Page(object):
```

Comment:
This is an extremely rudimentary stand-in for a real browser. I have a [separate branch](https://github.com/cbguder/merou/tree/selenium) that will hopefully replace this with Selenium at some point.